src/patterns/exit-the-page-quickly/index.md

```diff
@@ -0,0 +1,139 @@
+- to return to the service, they can search for the site they were using, or find it in their browser history
+- there are other things they can do to stay safe online – include a link to a ‘safety content page’ where they can learn more
+
+## Safety content page
+
+Create a page to give users important information they need to stay safe online.
+
+Generally, this page should tell the user other steps they can take to protect their privacy.
+
+What you include will depend on your service and who uses it. But wherever practical, you should recommend that the user takes some basic steps such as:
+- only using your service on a public device, such as in a library
+- using private browsing
+- clearing their browsing data and cookies
+
+Example of a safety content page:
+
+<hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+
+## Ways to stay safe online
+
+It can be easy for someone to see what you’re doing online, especially if you’re using a home computer.
+
+You can make it more difficult for them by taking some precautions.
+
+If you think your devices or internet search activities are being monitored, try to use a device that is not being monitored. That should be a device that the person does not or has not had physical or remote access to. This is the safest thing to do if you do not want someone to know that you’re visiting certain websites.
+
+### Cover your tracks online
+
+Every time you use the internet, your browser stores a record of where you’ve been online with:
+- a list of the pages you’ve looked at and the files you’ve downloaded
+- small files called cookies which remember your settings for different sites
+
+### Deleting browser history
+You can improve your safety by deleting your browser history. But you need to be careful because:
+- deleting cookies will also get rid of stored passwords for online accounts
+- clearing your history could make someone more suspicious
+
+Try to only remove information about the websites you want to keep private. You can find out how to do that on:
+- [Google Chrome](https://support.google.com/chrome/answer/95589?hl=en&rd=1)
+- [Internet Explorer](https://support.microsoft.com/en-us/windows/view-and-delete-your-browsing-history-in-internet-explorer-098ffe52-5ac9-a449-c296-c735c32c8678)
+- [Mozilla Firefox](https://support.mozilla.org/en-US/kb/delete-browsing-search-download-history-firefox)
+- [Safari](https://support.apple.com/kb/PH19215?locale=en_US&viewlocale=en_US)
+- [Opera](http://www.opera.com/help/tutorials/security/sharing/)
+
+### Browsing in private
+
+You can also look at a website without information like cookies being stored by using the private browsing setting in your browser. Go to the menu, click on ‘File’ and choose:
+
+- InPrivate on Internet Explorer
+- Private Browsing on Mozilla Firefox or Opera
+- Incognito on Google Chrome
+- Private Window on Safari
+
+<hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
```

Comment:
As per https://github.com/alphagov/govuk-design-system/issues/2631, we'll want to show this template as either:
- an coded example window
- a separate page